test/test_functions

```diff
@@ -28,6 +28,13 @@ CVMFS_TEST_S3_CONFIG=${CVMFS_TEST_S3_CONFIG:=}
 CVMFS_TEST_STRATUM0=${CVMFS_TEST_STRATUM0:=}
 CVMFS_TEST_SERVER_CACHE=${CVMFS_TEST_SERVER_CACHE:=}
 
+# This OPT variables are for the benchmark options under the benchmark_tests folder
+CVMFS_OPT_WARM_CACHE=${CVMFS_OPT_WARM_CACHE:=yes}
+CVMFS_OPT_ITERATIONS=${CVMFS_OPT_ITERATIONS:=1}
+CVMFS_OPT_TEST_TYPE=${CVMFS_OPT_TEST_TYPE:=callgrind} # callgrind / memcheck
+CVMFS_OPT_CONFIG_FILE=${CVMFS_OPT_CONFIG_FILE:=}
+CVMFS_OPT_OUTPUT_DIR=${CVMFS_OPT_OUTPUT_DIR:=~/cvmfs_benchmarks}
```

Comment:
I see.  In this case, I'd let the user decide via a `CVMFS_OPT_...` parameter which defaults to somewhere in /tmp.  I'm not sure if the tilde (~) is interpreted correctly in all cases.